integration-tests/typescript-fetch/src/generated/api.github.com.yaml/client.ts

```diff
@@ -320,11 +320,36 @@ import {
   AbstractFetchClient,
   AbstractFetchClientConfig,
   Res,
+  Server,
   TypedFetchResponse,
 } from "@nahkies/typescript-fetch-runtime/main"
 
+export class GitHubV3RestApiServers {
+  static default(): Server<"GitHubV3RestApi"> {
+    return "https://api.github.com" as Server<"GitHubV3RestApi">
+  }
+
+  static specific(url: "https://api.github.com") {
+    switch (url) {
+      case "https://api.github.com":
+        return "https://api.github.com" as Server<"GitHubV3RestApi">
+    }
+  }
+
+  static custom(url: string): Server<"GitHubV3RestApiCustom"> {
+    return url as Server<"GitHubV3RestApiCustom">
+  }
+
+  static reposUploadReleaseAsset(url: "https://uploads.github.com") {
```

Comment:
todo: risk of naming collisions between operation ids and `specific` / `custom` / `default`